File: Healthcare-app/ios/MDBRealmDoctor/MDBRealmDoctor/Condition.swift
//
//  Condition.swift
//  MDBRealmDoctor
//
//  Created by Karthick TM on 02/05/22.
//

import Foundation
import RealmSwift


class Condition: Object {
    @Persisted(primaryKey: true) var _id: ObjectId
    @Persisted var active: Bool?
    @Persisted var code: Codable_Concept?
    @Persisted var notes: String?
    @Persisted var subject: Reference?
    @Persisted var subjectIdentifier: String?
    
//    override init() {}
//
//    convenience init(_id: ObjectId, code: Codable_Concept?, notes: String?,subject: Reference? ) {
//        self.init()
//
//        self._id    = _id
//        self.code   = code
//        self.notes  = notes
//        self.subject = subject
//    }
}

File: Healthcare-app/android/MDBRealmPatient/app/src/main/java/com/wekanmdb/storeinventory/model/condition/Condition.kt
package com.wekanmdb.storeinventory.model.condition

import com.wekanmdb.storeinventory.model.embededclass.Codable_Concept
import com.wekanmdb.storeinventory.model.embededclass.Reference
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import org.bson.types.ObjectId

open class Condition(
    @PrimaryKey var _id: ObjectId = ObjectId(),
    var active: Boolean? = null,
    var code: Codable_Concept? = null,
    var notes: String? = null,
    var subject: Reference? = null,
    var subjectIdentifier: String? = null
) : RealmObject() {}